qmcpy/algorithms/distribution/__init__.py

```diff
@@ -18,15 +18,15 @@ def __init__(self, accepted_measures, trueD=None, distrib_data=None):
         super().__init__()
         # Abstract Properties
         self.distrib_data = distrib_data
-        self.trueD = trueD   
```

Comment:
I use PyCharm (I think I saw one of you using VSCode) and when I save a file in PyCharm it automatically applies my file standards (which in this case is remove trailing whitespace).  It's also standard for PEP8.  When I get lint in place I'll make a lot of edits like this.